### PR TITLE
dynamically update copyright year in footer

### LINKED ISSUE #83: update the copyright year in site footer
`2023 -> 2024`

### PR DESCRIPTION
**Pull request by dextrot**
This PR fixes #83

image

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -49,9 +49,13 @@
...
 master_doc = "index"
 
+from datetime import date
+
+year = date.today().year
+
 # General information about the project.
 project = "gliderpy"
-copyright = "2023, Filipe Fernandes"
+copyright =  str(year) + ", Filipe Fernandes"
```
Comment (dextrot) [inline docs/source/conf.py:58]:
```suggestion
copyright = str(year) + ", Filipe Fernandes"
```

**Comment by dextrot** [comment]
Hi @ocefpaf 
I updated the copyright year from `2023` to 2024 :smile: in the site footer

File: docs/source/conf.py [hunk 2]
```diff
@@ -49,9 +49,13 @@
...
 master_doc = "index"
 
+from datetime import date
+
+year = date.today().year
+
 # General information about the project.
 project = "gliderpy"
-copyright = "2023, Filipe Fernandes"
+copyright = str(year) + ", Filipe Fernandes"
```
Comment (ocefpaf) [inline docs/source/conf.py:58]:
```suggestion
copyright = f"{year}, Filipe Fernandes"
```

Comment (dextrot) [reply docs/source/conf.py]: Thanks for the suggestion. commited.

**Pull request merged**